api-tests/core/content-manager/api/basic-compo.test.api.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-const { createTestBuilder } = require('../../../../../../test/helpers/builder');
-const { createStrapiInstance } = require('../../../../../../test/helpers/strapi');
-const { createAuthRequest } = require('../../../../../../test/helpers/request');
+const { createTestBuilder } = require('../../../../test/helpers/builder');
```

Comment:
can't wait to have typescript and use typescript paths to avoid big relative imports.

Reply:
You can't really do that with typescript except if you bundle sadly. But I plan on making it an internal package to be able to require them :)